build.py

```diff
@@ -29,7 +29,12 @@
 parser.add_argument("--wasm", action="store_true", help="Build the WebAssembly files")
 parser.add_argument("--npm", action="store_true", help="Build the npm package")
 parser.add_argument("--play", action="store_true", help="Build the web playground")
-parser.add_argument("--samples", action="store_true", help="Compile the Q# samples")
+parser.add_argument(
+    "--samples",
+    action=argparse.BooleanOptionalAction,
+    default=True,
+    help="Build the samples (default is --samples)",
+)
```

Comment:
I'm being super nitpicky here but I'd prefer if this was moved below next to the other `BooleanOptionalAction` args.

Also I kinda liked the original description better... ("Compile the Q# samples (default is --samples)")